crates/cubecl-spirv/src/extensions/GLSL_std_450.rs

```diff
@@ -0,0 +1,91 @@
+/* automatically generated by rust-bindgen 0.70.1 */
+
+pub const GLSLstd450Version: ::std::os::raw::c_int = 100;
+pub const GLSLstd450Revision: ::std::os::raw::c_int = 3;
+#[repr(u32)]
```

Comment:
I've had multiple typos, and it's better to rely on the official header to avoid any mistakes. SPIR-V doesn't have builtin functions for any of this, so you need to use the `GLSL.std.450` extension if running on Vulkan, or `OpenCL.std` if running on OpenCL. That's why they're on the `SpirvTarget` trait.